packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -53,6 +54,18 @@ Provides: foreman-plugin-remote_execution
 A plugin bringing remote execution to the Foreman, completing the config
 management functionality with remote management functionality
 
+%package cockpit
+BuildArch:  noarch
+Requires:   cockpit
+Requires(post): systemd-sysv
```

Comment:
I wonder why this is needed. I thought it's only for legacy sysv init scripts but this is a native systemd service.

Reply:
Because we have it in foreman.spec: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/foreman/foreman/foreman.spec#L52 seems like it should not be there as well. Updated this spec